Data/Comtrade/get_comtrade.py

```diff
@@ -31,133 +29,258 @@ def nested_list(original_list, list_length):
     return nested_list
 
 
+# Premium subscription authorization code. Look this up in our
+# Comtrade account info page.
+# (https://comtrade.un.org/db/sysLoginAccess.aspx)
+auth_code = ""

```

Comment:
If we add .env file, then can pull auth code from there similar to:
```suggestion
auth_code = os.getenv("COMTRADE_AUTH_KEY")

```